src/round.rs

```diff
@@ -239,19 +233,6 @@ where
 /// See: [`DurationRound`]
 #[derive(Debug, Clone, PartialEq, Eq, Copy)]
 pub enum RoundingError {
-    /// Error when the Duration exceeds the Duration from or until the Unix epoch.
-    ///
-    /// ``` rust
-    /// # use chrono::{DurationRound, Duration, RoundingError, TimeZone, Utc};
-    /// let dt = Utc.with_ymd_and_hms(1970, 12, 12, 0, 0, 0).unwrap();
-    ///
-    /// assert_eq!(
-    ///     dt.duration_round(Duration::days(365)),
-    ///     Err(RoundingError::DurationExceedsTimestamp),
-    /// );
-    /// ```
-    DurationExceedsTimestamp,
```

Comment:
This enum is part of our public API, so `DurationExceedsTimestamp` can't be removed. That would be a breaking change.

Reply:
I would only update the doc of `DurationExceedsTimestamp`.